### ISSUE TITLE
The real-time progress of uploaded files is not displayed on macOs, but on windows

### ISSUE THREAD

**Issue by larkzhang** [kingToolbox/WindTerm#1961]
The real-time progress of uploaded files is not displayed when used in macOs. The real-time progress can be displayed on windows system. Is this issue solved in version 2.6

**Comment by kingToolbox** [comment]
Yes, it should have been fixed. Please wait for the release of the official `2.6.0` version, then download and try it out.

**Comment by larkzhang** [comment]
> 是的，它应该已经修复。请等待正式版发布，然后下载并试用。`2.6.0`

This problem still exists in the 2.6.0 official version
